Olha_Lesiuk/task_2.py

```diff
@@ -0,0 +1,15 @@
+a = 2
+a *= 2
+print(a)
+
+a = 2
+y = 2
+print(a * y)
+
+a = 2
+a /= 2
+print(a)
+
+a = 2
+y = 2
+print(a / y)
```

Comment:
use byte operations for / and * as second variant

Reply:
![image](https://user-images.githubusercontent.com/25909810/187702741-5e8a54d2-eeb6-431b-ace4-12114ffe2985.png)
No empty line at the end